src/models/freak.js

```diff
@@ -19,3 +21,19 @@ export const FreakModelDefault = {
   [FreakModelKeys.norm]: '',
   [FreakModelKeys.skills]: [],
 };
+
+const skillShape = {
+  value: PropTypes.string,
+  name: PropTypes.string,
+};
```

Comment:
I suppose for now it will have to do, but when we do a file for Skills to define their structure and everything else, we should extract the shape there.